clang/test/SemaCXX/cxx23-ctad-inherited-ctors.cpp

```diff
@@ -0,0 +1,260 @@
+// RUN: %clang_cc1 -fsyntax-only -std=c++23 -verify %s
```

Comment:
nit: you need to adjust the diagnostic messages after the https://github.com/llvm/llvm-project/commit/c7bfc41860a6abe5c92dc5afb47348b0c9e69963

Reply:
Thanks, fixed